apps/studio/src/server/modules/resource/resource.service.ts

```diff
@@ -609,14 +627,8 @@ export const publishPageResource = async ({
     })
   })
 
-  // Step 2: Trigger a publish of the site
-  await publishSite(logger, {
-    siteId,
-    userId: user.id,
-    resourceId,
-    isScheduled,
-    addCodebuildJobRow,
-  })
+  // Step 2: Trigger a publish of the site, if startSitePublish is true
+  if (startSitePublish) await publishSite(logger, siteId)
```

Comment:
question: why not just return `codeBuildId` here since the immediate next step is to grab `codeBuildId` from the codebase (and it's safe to assume this variable doesn't change)